src/Entity/SmsDeliveryReport.php

```diff
@@ -18,14 +20,19 @@
  *   id = "sms_report",
  *   label = @Translation("SMS Delivery Report"),
  *   base_table = "sms_report",
+ *   revision_table = "sms_report_revision",
  *   entity_keys = {
  *     "id" = "id",
  *     "uuid" = "uuid",
+ *     "revision" = "vid",
+ *     "langcode" = "langcode",
```

Comment:
'langcode' field is provided by the parent class `ContentEntityBase::baseFieldDefinitions()`. Question is: should the entity be translatable?